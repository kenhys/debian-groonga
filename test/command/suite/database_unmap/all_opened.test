table_create Users TABLE_HASH_KEY ShortText
column_create Users name COLUMN_SCALAR ShortText

load --table Users
[
{"_key": "alice",   "name": "Alice"},
{"_key": "bob",     "name": "Bob"},
{"_key": "charlie", "name": "Charlie"}
]

#@disable-logging
thread_limit 1
#@enable-logging
database_unmap

dump

