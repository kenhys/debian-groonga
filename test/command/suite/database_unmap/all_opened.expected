table_create Users TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Users name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table Users
[
{"_key": "alice",   "name": "Alice"},
{"_key": "bob",     "name": "Bob"},
{"_key": "charlie", "name": "Charlie"}
]
[[0,0.0,0.0],3]
database_unmap
[[0,0.0,0.0],true]
dump
table_create Users TABLE_HASH_KEY ShortText
column_create Users name COLUMN_SCALAR ShortText

load --table Users
[
["_key","name"],
["alice","Alice"],
["bob","Bob"],
["charlie","Charlie"]
]
